config/base/params.env

```diff
@@ -11,3 +11,4 @@ IMAGES_MOVERESULTSIMAGE=registry.access.redhat.com/ubi8/ubi-micro:8.8
 IMAGES_MARIADB=registry.redhat.io/rhel8/mariadb-103:1
 IMAGES_OAUTHPROXY=registry.redhat.io/openshift4/ose-oauth-proxy@sha256:ab112105ac37352a2a4916a39d6736f5db6ab4c29bad4467de8d613e80e9bb33
 ZAP_LOG_LEVEL=info
+MAX_CONCURRENT_RECONCILES=20
```

Comment:
match this to the unspecified default: https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/controllers/config/defaults.go#L113